packages/compiler/src/experimental/typekit/kits/type.ts

```diff
@@ -33,7 +33,7 @@ export interface TypeTypekit {
    */
   finishType(type: Type): void;
   /**
-   * Checks if a type is decorated with @error
+   * Checks if a type is decorated with \@error
```

Comment:
```suggestion
   * Checks if a type is decorated with `@error`
```

usually just do that